src/Advection/weno_default_stencils.jl

```diff
@@ -0,0 +1,685 @@
+using Printf
+
+@inline getvalue(i, j, k, grid, ψ, args...) = @inbounds ψ[i, j, k]
+@inline getvalue(i, j, k, grid, ψ::Function, args...) = ψ(i, j, k, grid, args...)
```

Comment:
don't we have an `identity` function that does this same thing?

Reply:
https://github.com/CliMA/Oceananigans.jl/blob/427c92fac963ee0883fa2766df15ba6a48e62736/src/Operators/interpolation_utils.jl#L39-L41